diff --git a/node_modules/react-codemirror2/.ts/index.js b/node_modules/react-codemirror2/.ts/index.js
index f007955..1bd10ff 100644
--- a/node_modules/react-codemirror2/.ts/index.js
+++ b/node_modules/react-codemirror2/.ts/index.js
@@ -389,7 +389,7 @@ var Controlled = (function (_super) {
             this.props.editorDidMount(this.editor, this.editor.getValue(), this.initCb);
         }
     };
-    Controlled.prototype.componentWillReceiveProps = function (nextProps) {
+    Controlled.prototype.UNSAFE_componentWillReceiveProps = function (nextProps) {
         if (SERVER_RENDERED)
             return;
         var preserved = { cursor: null };
@@ -511,7 +511,7 @@ var UnControlled = (function (_super) {
             this.props.editorDidMount(this.editor, this.editor.getValue(), this.initCb);
         }
     };
-    UnControlled.prototype.componentWillReceiveProps = function (nextProps) {
+    UnControlled.prototype.UNSAFE_componentWillReceiveProps = function (nextProps) {
         if (this.detached && (nextProps.detach === false)) {
             this.detached = false;
             if (this.props.editorDidAttach) {
diff --git a/node_modules/react-codemirror2/index.js b/node_modules/react-codemirror2/index.js
index 1f37fce..7f336d2 100644
--- a/node_modules/react-codemirror2/index.js
+++ b/node_modules/react-codemirror2/index.js
@@ -540,7 +540,7 @@ var Controlled = function(_super) {
     }
   };

-  Controlled.prototype.componentWillReceiveProps = function(nextProps) {
+  Controlled.prototype.UNSAFE_componentWillReceiveProps = function(nextProps) {
     if (SERVER_RENDERED) return;
     var preserved = {
       cursor: null
@@ -704,7 +704,7 @@ var UnControlled = function(_super) {
     }
   };

-  UnControlled.prototype.componentWillReceiveProps = function(nextProps) {
+  UnControlled.prototype.UNSAFE_componentWillReceiveProps = function(nextProps) {
     if (this.detached && nextProps.detach === false) {
       this.detached = false;

